analyzer/src/main/kotlin/managers/Bundler.kt

```diff
@@ -253,18 +253,22 @@ class Bundler(
 
                 OkHttpClientHelper.HTTP_TOO_MANY_REQUESTS -> {
                     throw IOException(
-                        "RubyGems reported too many requests, see " +
+                        "RubyGems reported too many requests when requesting meta-data for gem '$name', see " +
```

Comment:
Commit message nit: Drop "was".

Reply:
Done, rephrased the sentence to make it easier to understand.